guides/it/Miro.md

```diff
@@ -0,0 +1,43 @@
+# Miro
+
+All the information below and lots more useful stuff can be found on the Made Tech [Miro training board](https://miro.com/app/board/o9J_lkgJ4BU=/) (this is an internal link to a Miro board)
+
+## Getting access to Miro
+### New starters
+All new starters after November 2021 are automatically given a Free Restricted Miro licence when they join and are added to the Made Tech Team. This gives them the ability to view and comment with the option to request editing access.
+
+They can be added to any board or project in the Made Team Team by any other Made Tech Team member. To add them to another project or client Team, that Team Admin has to invite them.
+
+If they need access to edit then they can request it via Miro and a Miro Admin will be notified and grant that asap.
+
+### Been here a while
+Anyone from Made Tech can request a licence by emailing operations@madetech.com, or they can be invited by any other Made Tech Team member.
+
+The licence options are:
+Free Restricted - viewing and commenting (this is free)
+Full - editing access (this costs us money)
+
+### External users
+External users can't be added to the Made Tech team, they can only be added to separate client and project teams. The Team Admin for the client can invite them.
```

Comment:
Might also add an infosec line here reminding everyone of the rationale for this / importance of keeping client boards and data in properly separated environments?

Reply:
Updated and rationale added in the Teams section